pkg/karmadactl/addons/init/disable_option.go

```diff
@@ -15,6 +16,10 @@ type CommandAddonsDisableOption struct {
 	GlobalCommandOptions
 
 	KarmadaKubeClientSet *kubernetes.Clientset
+
+	WaitPodReadyTimeout int
```

Comment:
Please rebase the latest master branch, this flags has been removed, because the #3221 has been merged.